Pod/Classes/Core/RCTScrollView+EnclosingScrollView.m

```diff
@@ -164,6 +213,9 @@ - (void)sendScrollEventWithName:(NSString *)eventName
                                                       scrollViewZoomScale:self.scrollView.zoomScale
                                                                  userData:userData
                                                             coalescingKey:coalescingKey];
+
+  // static int transforms = 0;
+  // NSLog(@"eventses %d", transforms++);
   [[self valueForKey:@"_eventDispatcher"] sendEvent:scrollEvent];
```

Comment:
Just to be sure, does the `_eventDispatcher`, whatever type that may be, not already provide the ability to halt and event sending?

Reply:
doesn't seem to